frontend/src/scenes/session-recordings/sessionRecordingLogic.ts

```diff
@@ -30,6 +31,22 @@ export const parseMetadataResponse = (metadata: Record<string, any>): Partial<Se
     }
 }
 
+// TODO: Replace this with permanent querying alternative in backend. Filtering on frontend should do for now.
+const makeEventsQueryable = (events: SeekbarEventType[]): SeekbarEventType[] => {
+    return events.map((e) => {
+        let queryValue = e?.event ?? ''
+        if (['$pageview', '$pageleave'].includes(e.event)) {
```

Comment:
Should we pull in `eventToDescription` here and use that so we're definitely filtering by what's displayed? Maybe we do something similar for the event name too?

Reply:
good idea! especially now that we don't have to include bold JSX in `eventToDescription`